python/ray/dag/class_node.py

```diff
@@ -230,3 +230,99 @@ def num_returns(self) -> int:
             method = self._get_remote_method(self._method_name)
             num_returns = method.__getstate__()["num_returns"]
         return num_returns
+
+
+@DeveloperAPI
+class ClassMethodOutputNode(DAGNode):
+    def __init__(
+        self,
+        method_name: str,
+        method_args: Tuple[Any, ...],
```

Comment:
Annotate with more specific types like `Tuple[ClassMethodNode, int]`? Also why coupling them? Why not individual args?

Reply:
Ack on `Tuple[ClassMethodNode, int]`.
For `dag = a.inc_two.bind(inp, 1)`, I think `_bound_args = (inp, 1)`. Namely, individual args `inp` and `1` compose a tuple.